### DISCUSSION TITLE
fetch api nested data issue

### DISCUSSION THREAD

**Discussion by JhinAhri12** [nuxt/nuxt#16024] [Questions]
Hello all,
I have a problem with an async fetch

index page
```
<template>
    <section>
      <h2>Ses vidéos Youtube</h2>
      <div>
        <YoutubeChannel
        v-for="channel in channels"
                :key="id"
                :channel="channel"
        />
      </div>
      

    </section>
</template>
<script setup>
import logo from "@/assets/profil.webp"; 
import YoutubeChannel from '@/components/YoutubeChannel.vue'

const options = {
  method: 'GET',
  headers: {
    'X-RapidAPI-Key': '',
    'X-RapidAPI-Host': 'youtube-v3-alternative.p.rapidapi.com'
  }
};
const { data: channels } = await useAsyncData('get', () => $fetch('https://youtube-v3-alternative.p.rapidapi.com/channel?id=UCTwECeGqMZee77BjdoYtI2Q',options))

</script>
```

component YoutubeChannel 
```
<template>
    {{ channel.title }}
    <div v-for="index in 10" :key="index">
       {{ channel[index].title }}
       
    </div>
</template>

<script setup>
  const { channel } = defineProps(['channel'])
  JSON.parse(JSON.stringify(channel))
  console.log(channel)
</script>
```
![Screenshot_1](https://user-images.githubusercontent.com/31922998/210345549-7eea9462-1949-4aec-b167-4a74bad132f4.png)

I cant get the chanel[index].title and i dont know why, it looks like its on an object but i dont know why. If i write channel[index].title like in my exemple code i get an 500 error
Thx for any help

Regards.

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏
It seems like the issue is not about the actual API request but about how to access the nested data in the response so feel free to replace the API call with the actual data for the reproduction.

**Reply by JhinAhri12** [reply]
Hello i have try to reproduce the data here : 
https://stackblitz.com/edit/github-zg1ant?file=pages/index.vue

**Reply by TheAlexLichter** [reply]
Hey! Your stackblitz gives a 500 by default. Can you check again please?
